editNickname.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require_once "functions/allFunctions.php";
+
+if (!isset($_POST['id']) || !is_numeric($_POST['id'])){
+    header("location: index.php");
+    exit();
+}
+
+$db = connectToDB('pokemon-collection');
+$inputtedNewNickname = $_POST['newNickname'];
+$inputtedID = $_POST['id'];
+
+$cleanNickname = sanitiseText($inputtedNewNickname);
+$cleanID = sanitiseText($inputtedID);
```

Comment:
You could just run $_POST['id'] straight through sanitiseText

Reply:
Done